### DISCUSSION TITLE
Can I run 2 Slinger Servers for redundancy?

### DISCUSSION THREAD

**Discussion by blasco403** [GerryDazoo/Slinger#572] [Q&A]
Hello,

I run slinger from my house in Canada on a NUC windows server. I have an additional one not doing anything. I will be away for a long period of time. 

Can I run the slinger software on the two servers at the same time in case one of them has an issue?

Thanks.

**Comment by GerryDazoo** [comment]
Yes just make sure their port numbers are separated by at least two. 5

**Reply by blasco403** [reply]
Thanks Gerry,

Does it really matter if the ports are separated by 2 when using 2 different servers with different IP addresses? The ports per server should not matter. If on the same server running two different slinger instances I see why you would need them separated by 2.

I was playing after I asked the question and got the following running and configured.

Each server is running the same slinger software and configuration (I copied from one to the other without changing anything).

I have 2 slingboxes configured on each server (a 350 and a 500)

I currently have my Stream URL Shortcuts (internal and external for streaming and remote) pointing to 1 server for my 350 and the other server for my 500.

I have 2 different port forwarding rules on my router for external streaming and remote. External port xxx48 points internally to one server on xxx48 and external port xxx49 points internally to the other server on xxx48.

If one server quits working, I just have to change my internal dns names to point to the working server for both slingboxes, and update my port forwarding rules to point to the same internal server. No changing any of my shortcut URLS on any of my devices.

Seems to be working great.

I really do appreciate all the work you do/have done to keep my slingboxes operational.

Thanks.

**Reply by GerryDazoo** [reply]
You're right if they servers are on different boxes it does not mater if the port numbers are the same.
